nss-day-cohort-38/bangazon-ecommerce-api-iris-station
# Ag hot dog dump

Fixes # Hot Dog Dump

# Fetching Instructions
```shell
git fetch --all
git checkout ag-hot-dog-dump
```

# Description
Please include a summary of the change and which issue is fixed.

# List of changes
TESTING INSTRUCTIONS
1. (if you wish to preserve your own data to return to for whatever reason)
	`mv db.sqlite3 db.sqlite3.bak`
2. migrate to create a new fresh, blank db
	`python manage.py migrate`
3. build out the hot-dog heavy Andrew db:
	`python manage.py loaddata ecommerceapi/fixtures/fixtures.json`
4. Check out in the browser!
5. (after testing, if you wish to return to your old db)
	`mv db.sqlite3.bak db.sqlite3`
- 
- 

# Checklist:
